core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java

```diff
@@ -122,33 +121,39 @@ public MessageActionDTO handleAction(UserDTO actingUser, Message messageEntity,
 			}
 		}
 
-		private MessageActionDTO handleAction_Accept(UserDTO acceptingUser, DiscloseRequestMessage discloseRequestMessageEntity,
+		private MessageActionDTO handleAction_Accept(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
 				MessageActionDTO payload)
 		{
-			return updateResult(acceptingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_ACCEPTED,
+			return updateResult(actingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_ACCEPTED,
 					payload.getProperty("message"));
 		}
 
-		private MessageActionDTO handleAction_Reject(UserDTO rejectingUser, DiscloseRequestMessage discloseRequestMessageEntity,
+		private MessageActionDTO handleAction_Reject(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
 				MessageActionDTO payload)
 		{
-			return updateResult(rejectingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_REJECTED,
+			return updateResult(actingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_REJECTED,
 					payload.getProperty("message"));
 		}
 
-		private MessageActionDTO updateResult(UserDTO respondingUser, DiscloseRequestMessage discloseRequestMessageEntity,
+		private MessageActionDTO updateResult(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
```

Comment:
I know the method name is not changed, but still: This method name is not very logical to me. What about ``buildReturnValue``?